### PR TITLE
Examples tx 0, 1 and 2

### PR DESCRIPTION
**Pull request by SantiagoDevRel**
closing #5732

### CODE REVIEW

File: docs/docs/guides/wallet/transactions.md [hunk 1]
```diff
@@ -7,6 +7,124 @@ sidebar_label: 'Tutorial: Sending Transactions'
...
+  const gasPrice = await web3.eth.getGasPrice();
+
+  const tx = {
+    from: sender,
+    to: recipient,
+    value,
+    nonce,
+    gas,
+    gasPrice,
+  };
```
Comment (avkos) [inline docs/docs/guides/wallet/transactions.md:44]: We have defaultTransactionType: '0x2'

So if you want to sent legacy tx. set type: 0

**Review by avkos — CHANGES_REQUESTED** [review]
I think we already have example of all tx types. The question was to show how to send contract transaction with different types. For ex. Transfer ERC20 token using tx 0,1,2

**Comment by SantiagoDevRel** [comment]
@avkos this one looks better now!💪

File: docs/docs/guides/wallet/transactions.md [hunk 2]
```diff
@@ -7,6 +7,193 @@ sidebar_label: 'Tutorial: Sending Transactions'
...
+
+  console.log(txReceipt.transactionHash);
+}
+
+transfer();
+```
+
+## Transaction Type 1 (EIP-2930)
+
+This EIP was introduced in April 2021, it introduces a feature called 'Transaction Type and Access List.' This improvement allows saving gas on cross-contract calls by declaring in advance which contract and storage slots will be accessed.
```
Comment (jdevcs) [inline docs/docs/guides/wallet/transactions.md]:
```suggestion
This EIP was introduced in April 2021, it introduces a feature called 'Access List.' This improvement allows saving gas on cross-contract calls by declaring in advance which contract and storage slots will be accessed.
```

transaction type envelop was introduced in https://eips.ethereum.org/EIPS/eip-2718

File: docs/docs/guides/wallet/transactions.md [hunk 3]
```diff
@@ -7,6 +7,193 @@ sidebar_label: 'Tutorial: Sending Transactions'
...
+    type: 1,
+    accessList: [
+      {
+        address: contractAddress2, //contract1 is calling contract2
+        storageKeys: [
+          "0x0000000000000000000000000000000000000000000000000000000000000000",
+          "0x0000000000000000000000000000000000000000000000000000000000000001",
+        ],
+      },
+    ],
```
Comment (jdevcs) [inline docs/docs/guides/wallet/transactions.md]: I think also mention here that users can use https://docs.web3js.org/libdocs/Web3Eth#createaccesslist for getting accesslist of Tx

Comment (SantiagoDevRel) [reply docs/docs/guides/wallet/transactions.md]: amazing, i didn't know that, hanks for sharing @jdevcs

Comment (SantiagoDevRel) [reply docs/docs/guides/wallet/transactions.md]: Please verify i added it correctly in the last commit @jdevcs

**Comment by jdevcs** [comment]
> I think we already have example of all tx types. The question was to show how to send contract transaction with different types. For ex. Transfer ERC20 token using tx 0,1,2

lgtm, except Oleksii suggestion.

**Review by jdevcs — APPROVED** [review]

**Review by avkos — APPROVED** [review]

**Pull request closed without merging**